wpimath/src/main/native/include/units/angular_velocity.h

```diff
@@ -46,8 +46,8 @@ UNIT_ADD(angular_velocity, radians_per_second, radians_per_second, rad_per_s,
          unit<std::ratio<1>, units::category::angular_velocity_unit>)
 UNIT_ADD(angular_velocity, degrees_per_second, degrees_per_second, deg_per_s,
          compound_unit<angle::degrees, inverse<time::seconds>>)
-UNIT_ADD(angular_velocity, revolutions_per_minute, revolutions_per_minute, rpm,
```

Comment:
Removing `_rpm` is a breaking change. Can it be kept?

Reply:
Does anyone actually use it?  It can definitely be kept if you want it.

Reply:
It bugs me that it's named inconsistently w.r.t "turns" vs "revolutions" vs "rotations" though.  I'll leave it in for now, but can we please standardize for 2023 or smth?

Reply:
RPM is a common unit in datasheets. Having a literal for that is useful.